README.md

```diff
@@ -1 +1,8 @@
-# committee-tasks
\ No newline at end of file
+# Committee Tasks
+
+This repository collects the tasks which come out of the All Committees Meetings.
+
+See the full list of [issues](https://github.com/srobo/committee-tasks/issues) to see the lists of actions and their progress.
+
+This repo's issues are aimed at members of the volunteer committees.
```

Comment:
Suggestion: It might be nice to link out to what a "committee" is, perhaps from the ops manual

Reply:
7fda96d1a45fc61d9a3350a258256d0a82cd4532